app/internal/load_jokes.py

```diff
@@ -0,0 +1,38 @@
+import json
+from typing import Dict, List, Optional
+
+from app.database.models import Joke
+
+from sqlalchemy.orm import Session
+
+
+def get_jokes_from_json() -> List[Dict[str, Optional[str]]]:
+    """Reading all jokes from the a JSON file that have been """\
+        """copied from 'http://api.icndb.com/jokes'"""
+    try:
+        with open('app/resources/jokes.json', 'r') as f:
+            jokes_list = json.load(f)
+    except (IOError, ValueError):
+        return []
+    return jokes_list
+
+
+def add_jokes_to_db(session: Session) -> None:
+    """This function reads the jokes and inserts them into the db"""
+    all_jokes = get_jokes_from_json()
+    jokes_objects = [
+        Joke(text=joke['text'])
+        for joke in all_jokes
+        ]
```

Comment:
Can be in a single line

Reply:
Done

Reply:
?

Reply:
Something is wrong. I have changed everything you asked for and in my computer it is deferent from here.
It happened the last time too.
I am going to check why it is happening,
Sorry

Reply:
Maybe it is because I pushed once 2 pr on the same branch

Reply:
I fixed that. It was exactly what I thought. The files from my computer didn't push here.
Any way. Now it is really done.
Sorry for your time.